src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -106,6 +108,7 @@ class CompetitionRobotContainer {
 
     SmartDashboard.putData("AutoMode", autoChooser);
 
+    configureIntake();
```

Comment:
I don't necessarily think we need to create a separate method to just instantiate the intake subsystem.